styles/components/pagination/_goTo.scss

```diff
@@ -0,0 +1,41 @@
+.usa-dt-pagination__go-to {
+    margin-left: rem(18);
+    @media(max-width: $small-screen) {
+        margin-left: 0;
+    }
+
+    input {
+        width: rem(48);
+        margin: 0 rem(6);
+        padding: rem(7) rem(10);
+    }
+
+    label {
+        font-size: rem(12);
+        font-weight: normal;
+    }
+
+    button {
+        color: $color-white;
+        font-size: rem(12);
+        font-weight: $font-bold;
+        transition: opacity 0.15s;
+        padding: rem(7) rem(15);
+        border: 0;
+        border-radius: rem(2);
+
+        background-color: $color-primary;
```

Comment:
The color for this guy is so different from the pagination color which is kind of teal-ish. Was this by design?

Reply:
Yes, this was based on the Broker mockup found [here](https://bahdigital.invisionapp.com/d/main#/console/14221544/296021489/preview).